### PR TITLE
Update Terms of Service

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Description

Follow up from #662

This PR updates the Terms of Service document for the Openverse API. The current version of the document is hosted here: https://api.openverse.engineering/terms_of_service.html and can be found in code here: https://github.com/WordPress/openverse-api/blob/v2.4.3/api/terms_of_service.html

The new ToS will be located along with the rest of our API dev documentation here: https://wordpress.github.io/openverse-api/terms_of_service.html (URL won't work until this is merged).

## :warning: ToS changes :warning:

This PR also includes changes to our ToS wording. A rough difference view between the two can be found here: https://www.diffchecker.com/gc9T5FrF

**Most notably**, the following has changed:
- References to "CC Catalog" and "CC API` have been changed to reference Openverse.
- References to the ["Creative Commons Master Terms of Use"](https://creativecommons.org/terms/) have been removed. As far as I could tell, WordPress Foundation did not have an equivalent.
- Replaced reference to "Openverse Trademark Policy" (likely was "Creative Commons Trademark Policy) with references to WordPress Foundation's Trademark Policy
- Various spacing changes

## Testing Instructions

1. `just sphinx-make`
1. `just sphinx-serve`
1. Change Line 17 in `swagger.py` to be `tos_url = "http://0.0.0.0:3001/terms_of_service.html"`
1. Visit http://localhost:8000 and click the "Terms of Service" link
1. Verify that you are indeed redirected to the locally served sphinx documentation

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by zackkrida — APPROVED** [review]
I'm happy to approve this now; regardless of any potential concerns, it's certainly _more accurate_ than the current, live document, which seems like a win.

File: api/docs/terms_of_service.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+BY USING THE OPENVERSE API, YOU ACKNOWLEDGE THAT YOU HAVE READ, UNDERSTOOD, AND AGREED TO BE BOUND BY THESE API TERMS OF USE (hereafter, “API Terms”).
+By using the Openverse API, you also represent that you have the legal authority to accept the API Terms on behalf of yourself and any party you represent in connection with your use of the Openverse API.
+If you do not agree to the API Terms, you are not authorized to use the Openverse API.
+
+## 2. Licensed Uses and Restrictions
+
+In connection with your use of the Openverse API:
+
+  - You must adhere to all rate limits, registration requirements, and comply with all requirements in the Openverse API documentation;
+  - You must not scrape the content in the Openverse Catalog;
```
Comment (sarayourfriend) [inline api/docs/terms_of_service.md:17]: Why is this?

File: api/docs/terms_of_service.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  - You must not use multiple machines to circumvent rate limits or otherwise take measures to bypass our technical or security measures;
+  - You must not operate in a way that negatively affects other users of the API or impedes WordPress's ability to provide its services;
```
Comment (sarayourfriend) [inline api/docs/terms_of_service.md:19]:
```suggestion
  - You must not operate in a way that negatively affects other users of the API or impedes the WordPress Foundation's ability to provide its services;
```

Maybe?

File: api/docs/terms_of_service.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  - You must respect and comply with the terms and conditions applicable to the content and metadata accessible through the Openverse API, including by providing proper attribution to CC-licensed works and respecting the terms and conditions of platforms hosting CC-licensed works.
+  - You must not, and the products you develop must not encourage others to, violate any laws, regulations, or rights of any other person or entity, including intellectual property rights.
```
Comment (sarayourfriend) [inline api/docs/terms_of_service.md:21]:
```suggestion
  - You must not, and the products you develop must not, encourage others to, violate any laws, regulations, or rights of any other person or entity, including intellectual property rights.
```

File: api/docs/terms_of_service.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+You are responsible for independently verifying whether you have the right to use the content and data and what terms and conditions apply to your use.
+
+## 4. Openverse Trademark Use
+
+If you develop an application or other product or service using the Openverse API, you must prominently indicate that it was made using the Openverse API but is not endorsed or certified by Openverse.
+You must comply with the [WordPress Foundation Trademark Policy](https://wordpressfoundation.org/trademark-policy/) for all use of the Openverse/WordPress logos and other trademarks.
+
+## 5. Fees
+
+To help ensure the sustainability of the Openverse project, Openverse reserves the right to charge fees for commercial uses of the Openverse API and/or for heavy usage that places more of a burden on WordPress's systems and technical staff.
```
Comment (sarayourfriend) [inline api/docs/terms_of_service.md:36]:
```suggestion
To help ensure the sustainability of the Openverse project, Openverse reserves the right to charge fees for commercial uses of the Openverse API and/or for heavy usage that places more of a burden on the financial supporters of the WordPress Foundation and the maintainers of its systems.
```

WordPress doesn't have any "staff" as far as I know? Just a clarification. I'm sure the legal folks will have a better way of saying this.

File: api/docs/terms_of_service.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+## 6. Changes to the Terms
+
+Openverse is actively developing its product vision and strategy and may limit access to, and/or change the terms that apply to, the Openverse API at any time as plans take shape.
+If we make changes that we feel are material, we will make reasonable efforts to update developers who we know are actively using the Openverse API.
+Your continued use of the Openverse API after new and/or revised API Terms are effective indicates that you have read, understood, and agreed to those terms.
```
Comment (sarayourfriend) [inline api/docs/terms_of_service.md:42]:
```suggestion
Your continued use of the Openverse API after new and/or revised API Terms are effective indications that you have read, understood, and agreed to those terms.
```

**Review by sarayourfriend — COMMENTED** [review]
Just some nit-picky things. Agreed with Zack though, it doesn't seem like we need to necessarily wait to get it live considering the existing one is wrong... but IANAL so :salt: :grimacing:

File: api/docs/terms_of_service.md (hunk 1 above)
Comment (zackkrida) [reply api/docs/terms_of_service.md:17]: `- You must not scrape the content in the Openverse Catalog`

I'm also not sure what this line means; is it trying to discourage making a bulk copy of _all_ the data supplied by the API?

File: api/docs/terms_of_service.md (hunk 3 above)
Comment (zackkrida) [reply api/docs/terms_of_service.md:21]:
```suggestion
  - You must not, and the products you develop must not, encourage others to violate any laws, regulations, or rights of any other person or entity, including intellectual property rights.
```

😁

Comment (sarayourfriend) [reply api/docs/terms_of_service.md:21]: Whoops, I think that's where I _meant_ to add the suggested comma but am too sleepy and went a few words too far! :sweat_smile:

File: api/docs/terms_of_service.md (hunk 1 above)
Comment (AetherUnbound) [reply api/docs/terms_of_service.md:17]: I am honestly not sure. The "catalog" itself isn't even public. I'm happy to delete this line 🤷🏼‍♀️

File: api/docs/terms_of_service.md (hunk 2 above)
Comment (AetherUnbound) [reply api/docs/terms_of_service.md:19]: Oops, that's much better!

File: LICENSE [hunk 6]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Creative Commons
+Copyright (c) 2022 WordPress Foundation
```
Comment (sarayourfriend) [inline LICENSE:3]: I think actually this should be "the contributors" to match Gutenberg and core's license text:

https://github.com/WordPress/gutenberg/blob/trunk/LICENSE.md

https://github.com/WordPress/wordpress-develop/blob/trunk/src/license.txt#L3

File: api/docs/terms_of_service.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
+# Openverse API Terms of Service
+
+_Effective as of 4 May 2022_
```
Comment (sarayourfriend) [inline api/docs/terms_of_service.md:3]:
```suggestion
_Effective as of 5 May 2022_
```

**Review by sarayourfriend — APPROVED** [review]
LGTM! I feel safe merging this like Zack suggested, nothing seems too risky and the existing one is bogus. We can update it once we get feedback from WPF folks.

**Pull request merged**